apps/dashboard/src/components/container/ContainerActionsDialog.vue

```diff
@@ -142,20 +147,36 @@ const handleSubmitContainer = handleSubmit(async (values) => {
     }).catch((err) => handleError(err, toast));
   } else {
     // Creating a container
-    await containerStore.createEmptyContainer(
+    const container = await containerStore.createEmptyContainer(
         values.name,
         values.public,
         values.owner.id
-    ).then((c: ContainerResponse) => {
-      emit('create:container', c);
-      closeDialog();
-      toast.add({
-        severity: 'success',
-        summary: t('successMessages.success'),
-        detail: t('successMessages.createContainer'),
-        life: 3000,
+    ).catch((err) => handleError(err, toast));
+
+    if(!container) return;
```

Comment:
When can this be the case? And why shouldn't it show some kind of error message?

Reply:
Outdated cause from the other PR